tests/benchdnn/dnnl_common.hpp

```diff
@@ -14,639 +14,1002 @@
 * limitations under the License.
 *******************************************************************************/
 
-#ifndef DNNL_COMMON_HPP
-#define DNNL_COMMON_HPP
+#include <algorithm> // for std::reverse and std::copy
```

Comment:
This file is broken in many ways, benchdnn doesn't build. This seems to be a result of a rebase error.

Reply:
Addressed